ehrapy/tools/nlp/_medcat.py

```diff
@@ -156,213 +46,313 @@ def update_cat_config(self, concept_db_config: Config) -> None:
         """
         self.concept_db.config = concept_db_config
 
-    def extract_entities_text(self, text: str) -> Doc:
-        """Extracts entities for a provided text.
-
-        Args:
-            text: The text to extract entities from
-
-        Returns:
-            A spacy Doc instance. Extract the entities using. doc.ents
-        """
-        return self.cat(text)
-
-    def print_cui(self, doc: Doc, table: bool = False) -> None:
-        """Prints the concept unique identifier for all entities.
+    def set_filter_by_tui(self, tuis: list[str] | None = None):
+        """Restrict results of annotation step to certain tui's (type unique identifiers).
+        Note that this will change the MedCat object by updating the concept database config. In every annotation
```

Comment:
1 empty line after the summary. Always.
```suggestion

        Note that this will change the MedCat object by updating the concept database config. In every annotation
```